gen3/lib/utils.sh

```diff
@@ -390,3 +390,23 @@ check_terraform_module() {
   fi
   echo ${tversion}
 }
+
+save_logs_from_failed_pods() {
```

Comment:
expose this as a cli method - something like this:
```
gen3 save-failed-pod-logs folderName
```
, then from the Jenkins pipeline - call out to that method if `gen3 reset` has a failed exit code